### ISSUE TITLE
conversion issues

### ISSUE THREAD

**Issue by evilcats** [libgdx/fbx-conv#81]
Testing a few different input files and haven't been able to convert the following files:

Chair.FBX
WARNING:  Mesh contains more indices (50448) than the specified maximum (32767)
WARNING:  [Leathero2] Material doesn't extend FbxSurfaceLambert, replaced with RED diffuse

Eames.FBX
WARNING:  Mesh contains more indices (33792) than the specified maximum (32767)
WARNING:  [Wood] Material doesn't extend FbxSurfaceLambert, replaced with RED diffuse

Hut.FBX
WARNING:  [Mat_01] Material doesn't extend FbxSurfaceLambert, replaced with RED diffuse

Lightbulb.FBX
WARNING:  [Bulb] Node uses RrSs mode, transformation might be incorrect
WARNING:  [Bulb] Skipping empty node part, material: 'Black_Shiny.001'

Srt.FBX
WARNING:  Mesh contains more indices (260352) than the specified maximum (32767)

Hut.FBX
WARNING:  [Mat_01] Material doesn't extend FbxSurfaceLambert, replaced with RED diffuse

I'm running Debian 7 & fbx-conv version 0.01.0037 x64 (pre-release) using a precompiled binary;
Here is a link to an archive with the objects, and their respective command output's in text format.
https://www.dropbox.com/s/qsrdbhhiweqa2eh/fbxconv.rar?dl=0
Any help is much appreciated!

**Comment by xoppa** [comment]
What is the actual issue?

**Comment by evilcats** [comment]
the files listed above don't seem to be outputting a fbx file when I attempt to run them through the converter.

**Comment by xoppa** [comment]
fbx-conv doesnt output fbx files at all. https://github.com/libgdx/libgdx/blob/master/CONTRIBUTING.md#bug-reports

**Comment by nimeso** [comment]
@xoppa Hey ya! I'm the PHP guy on this project, @evilcats (Linux server guy) is not around so I'll try to do my best to explain :)

The converter is failing to produce G3DJ files from the fbx example files in the dropbox :( Whats strange is that we have another developer on a windows set up and he can convert them fine. But on our linux server we are getting indices count errors.

we keep getting WARNINGS but the G3DJ is not being created at all eg:

Chair.fbx
INFO:     FBX to G3Dx converter, version 0.01.0037 x64 (pre-release)
STATUS:   Loading source file
PROGRESS: Import FBX 100.00%                                                             
STATUS:   [shape(Box001)] Triangulating FbxMesh geometry
STATUS:   [shape(Line001)] Triangulating FbxMesh geometry
STATUS:   [shape(Box003)] Triangulating FbxMesh geometry
VERBOSE:  [shape(Box001)] polygons: 6108 (18324 indices), control points: 3172
VERBOSE:  [shape(Line001)] polygons: 16816 (50448 indices), control points: 8488
WARNING:  Mesh contains more indices (50448) than the specified maximum (32767)
VERBOSE:  [shape(Box003)] polygons: 704 (2112 indices), control points: 360
WARNING:  [Leathero2] Material doesn't extend FbxSurfaceLambert, replaced with RED diffuse
Illegal instruction

Some .fbx files are working fine. but most are failing :( we have tried different parameters in the command to increase stuff eg: -m 512000 but that still does not work :( 

Why would the windows version work fine but the linux version is failing?

I'll get @evilcats to write a better bug report when he around. 

Thanks for your time... any ideas would be great

**Comment by xoppa** [comment]
This is an issue tracker. If you need help then please use the forum or irc instead. If you are indeed reporting an issue (and thus have verified that it is indeed an issue with fbx-conv and not with your setup or models) then please be precisely when describing an issue. https://github.com/libgdx/libgdx/blob/master/CONTRIBUTING.md#using-the-issue-tracker

Btw, judging from the limited logs you've pasted, your models are not suitable for rendering. That said, although I don't have a linux machine to test on, I'm pretty sure that fbx-conv either creates/overwrites a file or reports an error or crashes. You still haven't been able to describe what the actual issue is you're reporting.

**Comment by nimeso** [comment]
@xoppa Thanks for your help.

I'll use the forum instead but I thought I had explained the issues :(
You will find detailed logs for each model in the .rar file @evilcats has sent you in dropbox
https://www.dropbox.com/s/qsrdbhhiweqa2eh/fbxconv.rar?dl=0

1. We have a windows developer that **can convert** all these models fine
2. They are **NOT** being converted on our Linux server :(
3. The G3DJ files are **NOT** being created at all
4. Why are they converting 100% correctly on the windows system BUT not Linux?

Can you convert the .fbx files in the dropbox rar at all? Do they work for you? 

Cheers heaps @xoppa I'm trying to give you as many details as I can :) that's why we provided the logs and models in the rar file for you.

**Comment by xoppa** [comment]
Please don't expect me to download some rar file to see a log. If the log is crucial for explaining the issue, then just include it in the issue. If it is too big to include (which i doubt it is) then include the relevant parts and link to a pastebin or gist for the full logs.

As said, fbx-conv should either create/overwrite a file, report an error or crash. So what is the actual issue? You say it doesnt create a file. The [code creating the file](https://github.com/libgdx/fbx-conv/blob/master/src/FbxConv.h#L147) is fairly straight forward. Does it report an error? Does it crash? Please precisely describe the issue you're reporting. Also describe the steps you took to verify that this is indeed an issue with fbx-conv and not with your setup.

**Comment by nimeso** [comment]
@xoppa @evilcats 

I can confirm that these model do convert on a windows system fine :( I just installed the converter on my windows system and they do work. 

Hmmmmm.... Something is not right on the Linux server but I have no idea what could be wrong We have installed all the libfbxsdk.so and libfbxsdk.dylib library's and **'some models'** are converting on the Linux server so I'm thinking that the library's are definitely working or else **'NO models'** would be getting converted. Like you, I don't have a local Linux system handy to test on. Very strange :(

Feel free to close this issue if you have no ideas about what could be going wrong. I really need to set up another Linux server on my local to test. I'm sorry @evilcats didn't really give you much info :( I will give him a good growling at for taking up your time. I'll get a far more detailed report for you when he gets back later today.
